index.html

```diff
@@ -178,36 +178,36 @@ <h1>Uber Navigation</h1>
                     <div class="connect">LET'S CONNECT</div>
                 </p>
                 <ul>
-                    <li><a href=""><img src="./images/github.png" /></a></li>
-                    <li><a href=""><img src="./images/linkedin.jpeg" /></a></li>
-                    <li><a href=""><img src="./images/sign.png" /></a></li>
-                    <li><a href=""><img src="./images/twiter.png" /></a></li>
-                    <li><a href=""> <img src="./images/Muliti.png" /></a></li>
+                    <li><a href=""><img src="./images/github.png" alt="here is the link to github"/></a></li>
+                    <li><a href=""><img src="./images/linkedin.jpeg" alt="here is the link to linkedin" /></a></li>
+                    <li><a href=""><img src="./images/sign.png" alt="here is the link to angellist" /></a></li>
+                    <li><a href=""><img src="./images/twiter.png" alt="here is the link to twitter" /></a></li>
+                    <li><a href=""> <img src="./images/Muliti.png" alt="here is the link to Muliti"/></a></li>
                 </ul>
 
                 <div class="get">Get my resume</div>
             </div>
             <div class="order-2">
                 <ul>
                     <li>
-                        <div class="lang-dropper">Languages <img src="./images/dropBtn.png" id="dropBtn1" alt="">
+                        <div class="lang-dropper">Languages <img src="./images/dropBtn.png" id="dropBtn1" alt="here is an drop-down icon">
                         </div>
                     </li>
                 </ul>
 
                 <div class="lang-flex">
-                    <div class="lang"><img src="images/js.png" alt="">javaScript</div>
-                    <div class="lang"><img src="images/html.png" alt="">HTML</div>
-                    <div class="lang"><img src="images/CSS.png" alt="">CSS</div>
+                    <div class="lang"><img src="images/js.png" alt="Here is javaScript logo">javaScript</div>
+                    <div class="lang"><img src="images/html.png" alt="here is html logo">HTML</div>
+                    <div class="lang"><img src="images/CSS.png" alt="here is a css logo">CSS</div>
                 </div>
 
                 <ul class="footer-list">
                     <li>
-                        <div class="frame">Frameworks <img src="./images/dropBtn2.png" id="dropBtn2" alt=""></div>
+                        <div class="frame">Frameworks <img src="./images/dropBtn2.png" id="dropBtn2" alt="here is an drop-down icon"></div>
                     </li>
                     <li>
                         <div class="frame"><span id="span">Skills</span><img src="./images/dropBtn2.png" id="dropBtn2"
-                                alt=""></div>
+                                alt="here is an drop-down icon"></div>
```

Comment:
- [ ] Please, consider the low contrast of the Contact me section for the header "Contact me" and for the paragraph below it. You can check it via [Wave](https://wave.webaim.org/) by adding your portfolio link https://anicetfantomas.github.io/setup-and-mobile-version-skeleton/
<img width="639" alt="Screen Shot 2022-03-02 at 15 59 48" src="https://user-images.githubusercontent.com/49064106/156357730-b45cfebb-85ff-44a1-afa0-2c455d85d8b6.png">

- [ ] Please, also consider the low contrast for the "Get it Touch" button of the contact form.
- [ ] Please, consider that your contact form control does not have a corresponding label
<img width="663" alt="Screen Shot 2022-03-02 at 16 02 59" src="https://user-images.githubusercontent.com/49064106/156358178-c199bd9b-2b79-469e-bd57-7fe0b3d4f51f.png">

Reply:
Thank you for your feedback  @elmar8287 

You are really doing a great job !
at this point about labels we only added them but they are empty because according to the figma design, there is no labels.

thank you again!!